types/wavesurfer.js/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for wavesurfer.js 5.0
+// Type definitions for wavesurfer.js 5.1
```

Comment:
use 5.1.0 instead

Reply:
Most type definitions reference the minor version, as patch versions shouldn't have interface changes in patch versions. My impression was that wavesurfer.js does adhere to semantic versioning. Does it not?